docs/sql-data-sources-parquet.md

```diff
@@ -432,4 +477,32 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
   </td>
   <td>3.1.0</td>
 </tr>
+<tr>
+  <td>parquet.encryption.column.keys</td>
```

Comment:
should we add non-Spark configs here? seems normally we don't do that.

Reply:
Yep, I've also noticed this.. Not sure, guess this is up to the community. Technically, it looks like an overhead to duplicate these parameters.